src/InverseSolve/iteratedTikhonov.jl

```diff
@@ -0,0 +1,51 @@
+export iteratedTikhonov
```

Comment:
Can you please add a minimal example for ```iteratedTikhonov``` and include it into ```test/runtests.jl```. You should be able to modify ```test/InverseSolve/testLeastSquares.jl```